### ISSUE TITLE
Configuration of metadata for year-elements

### ISSUE THREAD

**Issue by andre-hohmann** [kitodo/kitodo-production#3812]
Is it possible to configure the metadata of the year-element of newspapers? How is it possible? 

It should be possible to define the LABEL-metadata, as for example "Spielzeit 1983/1984" ([METS-Anwendungsprofil
](https://dfg-viewer.de/fileadmin/groups/dfgviewer/METS-Anwendungsprofil_2.3.1.pdf), page 12). Currently, only the ORDERLABEL-metadata can be used.

**Comment by matthias-ronge** [comment]
Related to #3741

**Comment by andre-hohmann** [comment]
@Kathrin-Huber: 

Yes, it seems to be still a problem. I cannot find a way to capture a value for the attribute LABEL for the year-level during the creation of the processes. Only the attribute ORDERLABEL is captured. 
Values captured in the field "Name des Jahres" are not regarded. See example below. 

It it possible to add the LABEL in the metadata editor after the creation of the process. From my point of view, this not user-friendly. 

Example
![image](https://user-images.githubusercontent.com/14835770/177089183-73fd318d-173e-40ec-b968-70d5c414fdb6.jpg)
